ui/raidboss/data/05-shb/dungeon/amaurot.js

```diff
@@ -242,25 +212,9 @@
         'The Face of the Beast': 'フェイス・オブ・ビースト',
         'Fallen Star': '流星',
         'Therion': 'メガセリオン',
-        'Terminus Twitcher': 'ターミナス・ツイッチャー',
```

Comment:
This is the same issue as #1485.  Because this is in a section with `missingTranslations` it is hard for me to know what is unused.  Please see my comment there.

Reply:
The `replaceSync` section is complete, all that is missing is on `replaceText`, I hesitated to delete `'missingTranslations': true,`, must i remove this line?
```
$ python util/find_missing_translations.py -f amaurot.js -l ja
.\ui\raidboss\data\05-shb\dungeon\amaurot.txt:47 text "Adds (N)"
.\ui\raidboss\data\05-shb\dungeon\amaurot.txt:48 text "Adds (SW)"
.\ui\raidboss\data\05-shb\dungeon\amaurot.txt:49 text "Adds (S)"
.\ui\raidboss\data\05-shb\dungeon\amaurot.js:22 "return {"
.\ui\raidboss\data\05-shb\dungeon\amaurot.js:30 "return {"
.\ui\raidboss\data\05-shb\dungeon\amaurot.js:38 "return {"
.\ui\raidboss\data\05-shb\dungeon\amaurot.js:63 "alertText: {"
.\ui\raidboss\data\05-shb\dungeon\amaurot.js:92 "alertText: {"
```

Reply:
What I'm trying to say is that in sections that have `missingTranslations: true`, please do not remove large amounts of translations.  If `missingTranslations: false` (because it's not listed), then `npm run test` will verify that the translations that are removed are truly unused.  However, in those sections that have `missingTranslations: true`, it is extra work for me to try to understand whether removed translations were truly unused or were mistakenly removed.

In this single case, you ran the script and I can see from the list of missing translations that the ones that are removed are not part of that list, so I can merge this PR.  However, in the future please do not remove large amount of translations from any section that has `missingTranslations: true`.

I hope that helps explain better!

Reply:
Oh yes, sorry. I try to do as many things as possible to help. sorry to give you so much concern.

Reply:
Ah no, it's the other way around.  Doing fewer things is easier for me to read.  When many things are done at once, it's much harder to follow.